src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -711,6 +711,14 @@ ENABLE_DRACONIAN_SECURITY_FEATURES = True
 ADD_CAPTCHA_FIELD_TO_FORMS = True
 {% endif %}
 
+{% if localsettings.RECAPTCHA_PUBLIC_KEY is defined %}
+RECAPTCHA_PUBLIC_KEY = {{ localsettings.RECAPTCHA_PUBLIC_KEY }}
```

Comment:
I think these two lines need to reference `RECAPTCHA_PUBLIC_KEY` now instead of `localsettings. RECAPTCHA_PUBLIC_KEY`